src/Sylius/Bundle/ThemeBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('sylius_theme');
+
+        $rootNode
+            ->addDefaultsIfNotSet()
+            ->fixXmlConfig('location')
+                ->children()
+                    ->arrayNode('locations')
+                        ->requiresAtLeastOneElement()
+                        ->performNoDeepMerging()
+                        ->defaultValue(['%kernel.root_dir%/themes'])
+                        ->prototype('scalar')
```

Comment:
`->end()` is not necessary as "closing open nodes", it only returns the parent node. Anyway, TDD for configurations and extensions seems great and it has to work correctly :)